packages/backend/src/utils/modelsUtils.ts

```diff
@@ -71,3 +71,14 @@ export async function deleteRemoteModel(machine: string, modelInfo: ModelInfo):
     console.error('Something went wrong while trying to stat remote model path', err);
   }
 }
+
+export function addModelPropertiesToEnvironment(modelInfo: ModelInfo, envs: string[]) {
```

Comment:
i would prefer a method that does not change the parameter it receive, and have something which return the environements variable.

So instead of having
```
addModelPropertiesToEnvironment(modelInfo, envs);
```

I would rather have

```
envs.push(...getModelPropertiesForEnvironment(modelInfo));
```